dvc/remote/local.py

```diff
@@ -799,3 +799,7 @@ def unprotect(path_info):
             RemoteLOCAL._unprotect_dir(path)
         else:
             RemoteLOCAL._unprotect_file(path)
+
+    @property
+    def is_copy_link_type(self):
+        return self.cache_types[0] == "copy"
```

Comment:
By default "self.cache_types" is going to be "reflink, copy", not "copy". Link type that is going to be used is determined in the `link()`.